src/main/resources/camel-server.xml

```diff
@@ -1,63 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
If we delete this xml, we should make implicit uri to broker:transportConnector in src/main/resources/spring/camel-server.xml.